module-2/lab-df-calculation-and-transformation/your-code/challenge-3.ipynb

```diff
@@ -26,7 +26,7 @@
   },
```

Comment:
Sub Problem 3 - Collecting Information: you should do with a conditional similar to the following:
a) customers with amount spent below 75 percentile would be labeled as normal
b) customers with amount spent between 75 and 95 percentile would be labeled as preferred
c) customers with amount spent above 95 percentile would be labeled as VIP
When doing >75 and >95 customers above 75 will also be included in >95. 
I'm sending you an example of solution.